README.md

```diff
@@ -1,2 +1,210 @@
-# asgardio-java-oidc-sdk
-OIDC Java SDK for Asgardio
+# Asgardio OIDC SDK for Java
+
+[![Build Status](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fwso2.org%2Fjenkins%2Fjob%2Fasgardio%2Fjob%2Fasgardio-java-oidc-sdk%2F&style=flat)](https://wso2.org/jenkins/job/asgardio/job/asgardio-java-oidc-sdk
+/) [![Stackoverflow](https://img.shields.io/badge/Ask%20for%20help%20on-Stackoverflow-orange)](https://stackoverflow.com/questions/tagged/wso2is)
+[![Join the chat at https://join.slack.com/t/wso2is/shared_invite/enQtNzk0MTI1OTg5NjM1LTllODZiMTYzMmY0YzljYjdhZGExZWVkZDUxOWVjZDJkZGIzNTE1NDllYWFhM2MyOGFjMDlkYzJjODJhOWQ4YjE](https://img.shields.io/badge/Join%20us%20on-Slack-%23e01563.svg)](https://join.slack.com/t/wso2is/shared_invite/enQtNzk0MTI1OTg5NjM1LTllODZiMTYzMmY0YzljYjdhZGExZWVkZDUxOWVjZDJkZGIzNTE1NDllYWFhM2MyOGFjMDlkYzJjODJhOWQ4YjE)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/wso2/product-is/blob/master/LICENSE)
+[![Twitter](https://img.shields.io/twitter/follow/wso2.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=wso2)
+---
+
+The Asgardio OIDC SDK for Java enables software developers to integrate OIDC based SSO authentication with Java Web
+applications. The SDK is built on top of the NimbusDS OAuth 2.0 SDK with OpenID Connection extensions library which
+ allows Java developers to develop cross-domain single sign-on and federated access control solutions with minimum
+  hassle.
+
+## Getting Started
+
+You can experience the end-to-end capabilities of Asgardio Java OIDC SDK by trying out the sample application
+ featured in Asgardio OIDC tomcat agent getting started guide [TODO link].
+
+In this section, we would be looking at how you can embed the Asgardio Java OIDC SDK to your existing Java web app.
+
+### Table of Contents
+- [Initializing the SDK](#initializing-the-sdk)
+  * [Maven dependency](#maven-dependency)
+  * [Configuration file](#adding-the-configuration-properties)
+  * [OIDC Manager](#oidc-manager)
+- [Login](#login)
+- [Handle Callback response](#handle-callback-response)
+- [Logout](#logout)
+- [Request Resolving](#request-resolving)
+
+### Initializing the SDK
+This section covers the preliminary steps needed for getting the SDK ready to be used in your java webapp.
+
+#### Maven dependency
+You can add the Asgardio Java OIDC SDK to your java project by installing it as a maven dependency:
+
+```xml
+<dependency>
+    <groupId>io.asgardio.java.oidc.sdk</groupId>
+    <artifactId>io.asgardio.java.oidc.sdk</artifactId>
+    <version>1.0.0</version>
```

Comment:
Change to 0.1.0-SNAPSHOT

Reply:
Resolved in https://github.com/asgardio/asgardio-java-oidc-sdk/pull/16/commits/ad5be6d648df3cff04ac42fde90ef9408e5597b0